avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -90,6 +93,109 @@
 
   // Connect -> Avro
 
+  @Test
```

Comment:
nit: these sections of tests generally are ordered simple -> complex. it'd be nice to just move these farther down in the file.